R/profile_plot.R

```diff
@@ -130,24 +131,31 @@ profile_plot <- function(mydir, rep, para, profilesummary) {
 
   dev.off()
 
-  maxyr <- min(profilesummary$endyrs)
+  maxyr <- min(profilesummary$endyrs + 1)
   minyr <- max(profilesummary$startyrs)
-
+  
```

Comment:
Extra white space (i.e., spaces not the carriage return) should be removed.

Reply:
Ignore formatting comments like this that I made earlier. We can always use a lintr later like @iantaylor-NOAA suggested.